test/e2e-helm/e2e_helm_suite_test.go

```diff
@@ -20,11 +20,11 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/onsi/ginkgo" //nolint:golint
-	. "github.com/onsi/gomega" //nolint:golint
```

Comment:
is it not so longer file with the upper version of the checker. Great to know. 👍